src/contexts/OptimizationLoadContext/index.js

```diff
@@ -44,12 +55,30 @@ export const OptimizationLoadProvider = ({ settings, children }) => {
     }
   }
 
+  const handleUpdateOptimizationConfigs = (configs) => {
```

Comment:
Please create a global method for this update using param like `configs`, in order to optimize the method